packages/calcite-components/src/tests/commonTests/utils.ts

```diff
@@ -74,3 +74,26 @@ export function getTagOrHTMLWithBeforeContent(componentTestSetup: TagOrHTML | Ta
 export function propToAttr(name: string): string {
   return name.replace(/([a-z0-9])([A-Z])/g, "$1-$2").toLowerCase();
 }
+
+export function validateCaretIndex({
```

Comment:
Can you move this util to [`tests/utils.ts`](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/tests/utils.ts) instead? `commonTests` is for the comprehensive, common test helpers and their respective internal modules.